lib/maplibre-compose/src/androidMain/kotlin/dev/sargunv/maplibrecompose/core/layer/HillshadeLayer.kt

```diff
@@ -13,27 +15,27 @@ internal actual class HillshadeLayer actual constructor(id: String, actual val s
   Layer() {
   override val impl = MLNHillshadeLayer(id, source.id)
 
-  actual fun setHillshadeIlluminationDirection(direction: Expression<Number>) {
+  actual fun setHillshadeIlluminationDirection(direction: IntExpression) {
```

Comment:
```suggestion
  actual fun setHillshadeIlluminationDirection(direction: FloatExpression) {
```

see https://github.com/maplibre/maplibre-native/blob/72d2c70130fcd4e320cf623f9bec811108b6e201/include/mbgl/style/layers/hillshade_layer.hpp#L48